backend/cmd/headlamp.go

```diff
@@ -594,6 +598,8 @@ func createHeadlampHandler(config *HeadlampConfig) http.Handler {
 
 	// Auth token management
 	r.HandleFunc("/auth/set-token", config.handleSetToken).Methods("POST")
+	// Expose user info so the frontend can show the current user in the top bar using the per-cluster auth cookie.
+	r.HandleFunc("/api/me", config.handleMe).Methods("GET")
```

Comment:
instead of `/api/me?cluster=clustername` it should be `/clusters/{clusterName}/me`
this way we can keep the original cookie path `"/clusters/" + cluster` without reducing security